.github/workflows/test.yml

```diff
@@ -19,29 +19,30 @@ jobs:
     - run: sudo apt-get update ; sudo apt-get install libmbedtls-dev
     - run: make ${{ matrix.target }}
   linux_bigendian:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
So apparently `ubuntu-latest` is misleading and is actually ubuntu 20.04(which seems to have qemu issues).